### PR TITLE
conda-smithy 3.30.2

### PR DESCRIPTION
**Pull request by beckermr**
Merge only after success.

This pull request was auto-generated by [rever](https://regro.github.io/rever-docs/)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ requirements:
     - python >=3.6
     - setuptools
     - conda >=4.2
-    - conda-build >=3.18.3
+    - conda-build >=3.28.2
```
Comment (mbargull) [inline recipe/meta.yaml:33]: Should be compatible with older versions too -- but I'm also fine to up this.

**Review by mbargull — APPROVED** [review]
Thanks for making the release!

File: recipe/meta.yaml (hunk 1 above)
Comment (beckermr) [reply recipe/meta.yaml:33]: yep sorry - fixing now.

Comment (beckermr) [inline recipe/meta.yaml:33]:
```suggestion
    - conda-build >=3.18.3
```

**Pull request merged**